charts/sk8s/values.yaml

```diff
@@ -16,6 +16,11 @@ functionController:
     repository: sk8s/function-controller
     tag: 0.0.1-SNAPSHOT
     pullPolicy: IfNotPresent
+  sidecar:
+    image:
+      repository: sk8s/function-sidecar
```

Comment:
repository and pullPolicy aren't used anywhere in the chart

Reply:
Fixed.